.github/workflows/llm-connectivity.yml

```
@@ -1,79 +1,88 @@
+          LLM_PROBE_MODEL: ${{ inputs.model || vars.MODEL1 }}
+        run: |
+          printf 'connectivity probe' > /tmp/p.txt
+          # endpoint 覆盖：输入非空时优先
+          [[ -n "$LLM_BASE_URL_OVERRIDE" ]] && export LLM_BASE_URL="$LLM_BASE_URL_OVERRIDE"
+          echo "probe target: $LLM_BASE_URL / $LLM_PROBE_MODEL"
+          reply=$(bash pipeline/metering/metering-wrapper.sh --model "$LLM_PROBE_MODEL" \
+            --prompt-file /tmp/p.txt --max-tokens 16 --temperature 0 --thinking disabled \
+            --role probe --tag connectivity --usage-compat-dir llm-usage)
+          echo "reply(≤200B): ${reply:0:200}"
+      - name: usage 记录入 run 摘要（BEH-09 留痕）
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:1:5: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)